ansible/deploy-all.yaml

```diff
@@ -1,35 +1,38 @@
 ---
 ############################################
+# Third-party
+############################################
+- include: infra-amqp-brokers.yaml
+  when: not parasitic
+
+- include: infra-deploy-irods-elasticsearch.yaml
+  when: not parasitic
+
+############################################
```

Comment:
It appears they were added only in the private repo; it presumably only works here because ansible-merge-tool copies everything together. So there's definitely something amiss, since this means we're publishing a playbook in the public repo which only we can run.